experiments/ddpg/HalfCheetah_v2.py

```diff
@@ -0,0 +1,43 @@
+from torchrl import registry
+from torchrl import utils
+from torchrl.problems import base_hparams, DDPGProblem
+from torchrl.agents import BaseDDPGAgent
+
+@registry.register_problem
+class DDPGHalfCheetah(DDPGProblem):
+  def __init__(self, *args, **kwargs):
+    self.env_id = 'HalfCheetah-v2'
```

Comment:
You can remove this constructor and move the `env_id` to hyperparameters. It is easier to extend a single Problem class to multiple environments.
```
params.env_id = 'HalfCheetah-v2'
```